documentation/1.11/content/_index.md

```diff
@@ -49,3 +49,19 @@ who wish to contribute to the WebLogic Image Tool code will find useful informat
 * [WebLogic Monitoring Exporter](https://github.com/oracle/weblogic-monitoring-exporter)
 * [WebLogic Logging Exporter](https://github.com/oracle/weblogic-logging-exporter)
 * [WebLogic Remote Console](https://oracle.github.io/weblogic-remote-console/)
+
+### Supported Fusion Middleware (FMW) products
+Use WebLogic Image Tool to create, update, and rebase the following FMW product images:
+
+* Oracle WebLogic Server (WLS)
+* Fusion Middleware Infrastructure (FMW)
+* Oracle BPM Suite for business process (SOA)
+* Oracle Service Bus (OSB)
+* Oracle Analytics Server (OAS)
+* Oracle Data Integrator (ODI)
+* Oracle Access Management (OAM)
+* Oracle Identity Governance (OIG)
+* Oracle Unified Directory (OUD)
+* Universal Content Management (UCM)
+* Oracle Managed File Transfer (MFT)
+* Oracle Internet Directory (OID)
```

Comment:
Did you leave these off intentionally?
* Oracle WebCenter Sites (WCS)
* Oracle Identity Manager (IDM)
* Oracle WebCenter Content (WCC)
* Oracle WebCenter Portal (WCP)

Reply:
@ddsharpe I did not. They were not listed in the JIRA from Monica. I'll add them now.

Reply:
Yes I missed those thanks Derek